grpc/protos/zoekt/webserver/v1/webserver.proto

```diff
@@ -119,23 +119,23 @@ message ListRequest {
 }
 
 message ListOptions {
+  reserved 16;
+
   enum RepoListField {
+    reserved 2;
```

Comment:
Do we need to add the field name to "reserved" as well? See [here](https://protobuf.dev/programming-guides/proto3/#fieldreserved)

Reply:
We don't use JSON serialization of these structs.